### PR TITLE
feat(HACBS-2225): add new utils

### PR DESCRIPTION
**Pull request by davidmogar**
As part of this commit, the Dockerfile is changed so it doesn't depend on a remote repository.

### CODE REVIEW

**Review by johnbieren — DISMISSED** [review]
I think `bash-utils` is a better name than just `utils` but if you don't think mentioning that they are bash, I suppose that is fine

File: utils/merge-json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env sh
+
```
Comment (mmalina) [inline utils/merge-json:2]: Could we have some basic description? Like what this is used for and possibly which task uses it.

Comment (davidmogar) [reply utils/merge-json:33]: Done.

File: utils/get-resource [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env sh
+
```
[the remark this answers is not in the document]
Comment (davidmogar) [reply utils/get-resource:21]: First done. README for a different PR maybe.

File: utils/get-resource [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+# Usage: kubectl_script.sh <resource_type> <namespaced_name> [jsonpath]
+#
+# Parameters:
+#   <resource_type>   - The Kubernetes resource type (e.g., pod, deployment, service).
+#   <namespaced_name> - The namespaced name of the resource in the form "namespace/name".
+#   [jsonpath]        - Optional JSONPath expression to filter the output.
+#
+# Requirements:
+#   - This script requires the 'kubectl' cli. Ensure that 'kubectl' is
+#     on yourinstalled
```
Comment (johnbieren) [inline utils/get-resource:19]: typo

**Review by johnbieren — DISMISSED** [review]

**Review by johnbieren — APPROVED** [review]

**Pull request merged**